eva_cttv_pipeline/trait_mapping/utils.py

```diff
@@ -0,0 +1,18 @@
+def request_retry_helper(function, retry_count: int, url: str):
```

Comment:
There is another function similar to this in one of OxO classes. Can that class use this function?